### ISSUE TITLE
flickering when playing mp4 file with video-sync=display-resample

### ISSUE THREAD

**Issue by unic0rn** [mpv-player/mpv#6600]
### mpv version and platform

0.29.1 - windows 10

### Reproduction steps

play the file provided with this config file:
```
quiet=yes
ao=wasapi
vo=gpu
video-sync=display-resample
interpolation=yes
tscale=triangle
priority=abovenormal
ytdl-format=bestvideo[height<=?1080]+bestaudio
msg-level=ffmpeg=fatal,ao=error,sub=error
```

with video-sync=audio everything works correctly.

### Expected behavior

proper playback.

### Actual behavior

weird flickering at certain moments.

now, the file provided is a smaller part of a larger file, originally recorded using windows game dvr. as i've mentioned above, with video-sync=audio (default), everything is fine. with video-sync=display-resample, flickering occurs. and now the most interesting part: the sample file was produced by remuxing part of the original mp4 file into new mp4 file with ffmpeg -t 00:25 -c:v copy -c:a copy. when remuxing to mkv instead of mp4, the resulting file plays correctly.

### Log file

https://aurora.irc.arloria.net/test.log

### Sample files

https://aurora.irc.arloria.net/test.mp4

**Comment by wm4** [comment]
Seems like an issue with vo_gpu interpolation. Maybe the enable/disable heuristic.

**Comment by Akemi** [comment]
i believe this was caused by the problem fixed with #8662. if the issue still persists please open a new issue.
